x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/lucene/LuceneSourceOperatorTests.java

```diff
@@ -93,16 +95,14 @@ private LuceneSourceOperator.Factory simple(DataPartitioning dataPartitioning, i
     }
 
     @Override
-    protected String expectedToStringOfSimple() {
-        assumeFalse("can't support variable maxPageSize", true); // TODO allow testing this
-        return "LuceneSourceOperator[shardId=0, maxPageSize=**random**]";
+    protected Matcher<String> expectedToStringOfSimple() {
+        return matchesRegex("LuceneSourceOperator\\[shardId=0, maxPageSize=\\d+]");
```

Comment:
Why don't we have to escape the closing brackets? Other usages of `matchesRegex` in our code base escape the closing brackets as well.
```suggestion
        return matchesRegex("LuceneSourceOperator\\[shardId=0, maxPageSize=\\d+\\]");
```
Applies to all of the regex added by this PR.

Reply:
IntelliJ warned me about closing these brackets as "unneeded" and I thought "oh, that makes it slightly more readable....."